dataikuapi/dss/ml.py

```diff
@@ -829,17 +829,28 @@ def set_range(self, min=None, max=None, nb_values=None):
 
     @property
     def range(self):
```

Comment:
Not sure about this, but maybe rename it to `range_settings` to be sure we do not confuse `Range` and `RangeSettings`

Reply:
I don't see the pro of renaming this property. To me the point is to still be able to write e.g. `rf_settings.n_estimators.range.min = 10` and also e.g. `rf_settings.n_estimators = Range(10, 100)`.

Reply:
That was before nesting the range inside `NumericalHyperparameterSetting`, resolving it